test/int/cluster_test.go

```diff
@@ -151,7 +151,7 @@ var _ = Describe("AtlasCluster", func() {
 
 				Expect(k8sClient.Update(context.Background(), createdCluster)).To(Succeed())
 
-				Eventually(testutil.WaitFor(k8sClient, createdCluster, status.TrueCondition(status.ReadyType), validateClusterCreatingFunc()),
+				Eventually(testutil.WaitFor(k8sClient, createdCluster, status.TrueCondition(status.ReadyType)),
```

Comment:
@vasilevp  this fixes the test flakiness as the `validateClusterCreatingFunc` expects that the cluster to be in good shape and here it isn't